packages/connect-thegraph/src/core/GraphQLWrapper.ts

```diff
@@ -17,6 +17,10 @@ export default class GraphQLWrapper {
   #verbose: boolean
 
   constructor(subgraphUrl: string, verbose = false) {
+    if (!subgraphUrl || !subgraphUrl.includes('http')) {
```

Comment:
Nitpick:

```suggestion
    if (!subgraphUrl || !subgraphUrl.startsWith('http')) {
```

Reply:
Haha yeah, was just gonna comment that. :)